database/src/main/java/com/pip/chatbot/repository/joke/JokesRepository.java

```diff
@@ -76,31 +72,30 @@ public JokesRepository(DSLContext dsl) {
     public Optional<Category> createTemporaryCategory(String category) {
         var result = dsl.insertInto(CATEGORY)
                 .set(JOKE.CATEGORY, category)
-                .set(JOKE.IS_CONFIRMED, !CONFIRMED)
+                .set(JOKE.IS_CONFIRMED, CONFIRMED)
                 .returning()
                 .fetchOne();
 
         return Optional.ofNullable(result.into(Category.class));
     }
 
-    public Optional<MarkApi> createMark(Mark mark) {
+    public Optional<Mark> createMark(Mark mark) {
 
         var result = dsl.insertInto(MARK)
                 .set(MARK.JOKE_ID, mark.getJokeId())
                 .set(MARK.MARK_, mark.getMark())
                 .returning()
```

Comment:
Create'y niech nie zwracają Optionala, bo to w tym wypadku jooq zawsze zwróci wartość lub poleci exception